.github/workflows/haskell.yml

```diff
@@ -390,6 +390,83 @@ jobs:
     - name: Run fourmolu
       run: ./scripts/fourmolize.sh
 
+  gen-cddl:
+    needs: build
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    strategy:
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v4
+
+    steps:
```

Comment:
I guess  this rogue ` steps` is causing the action to fail, since it was declared already a line above

Reply:
That's probably it, thanks!